code/js/constant/rcalc-constant-enum-file-path.js

```diff
@@ -0,0 +1,4 @@
+const filePath = {
```

Comment:
Let's agree on some syntax of enum so. All letters will be big, it will end of word CONST and for space between words will be underscore ... so 
FILE_PATH_CONST
... and name of file it can be without word enum
... my idea is: FilePathConstant.js / ConstantFilePath.js / file-path-constant.js / constant-file.path.js - ... choose your favorite ... and man do it for all contant files. :-)

Reply:
ok to keep similar style with the rest of my files I go for this style file-path-constant.js... also I like the syntaxing for enum FILE_PATH_CONST

Reply:
enum syntax used for all constant files